yaksh/models.py

```diff
@@ -1104,9 +1115,11 @@ class TestCase(models.Model):
 
 class StandardTestCase(TestCase):
     test_case = models.TextField(blank=True)
+    marks = models.FloatField(default=0.0)
```

Comment:
So each test case will have marks!
Instead I think a single constant fraction value for each testcase is a better option.
Else we can sum the total testcases and find the ratio of right to total.